models/content-type-snippets/index.ts

```diff
@@ -1 +1 @@
-export * from './seo_metadata';
+export * from './sEOMetadata';
```

Comment:
This file has weird name :D also cTAButton, check also the others :)

Reply:
That's because I set the file name resolver to camelCase which I think fits better into our codebase. Unfortunately, it doesn't look too well for names that start with an upper-cased word.

Reply:
I adjusted the resolver to use pascal case when the second letter is upper cased and camel case otherwise.